source/frontend/src/features/mapSideBar/property/MotiInventoryContainer.tsx

```diff
@@ -57,7 +58,8 @@ export const MotiInventoryContainer: React.FunctionComponent<
   const handleSaveClick = async () => {
     if (formikRef !== undefined) {
       formikRef.current?.setSubmitting(true);
-      formikRef.current?.submitForm();
+      await formikRef.current?.submitForm();
+      setIsValid(formikRef.current?.isValid ?? false);
```

Comment:
I see other places doing: 
```tsx
var result = await (formikRef.current?.submitForm() ?? Promise.resolve());
setIsValid(...)
```
why not use the same pattern here (awaiting the promise)?